components/starter-kits/starter-kits.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import Container from '../container/container';
+import Link from '../link/link';
+import Kits from './starter-kits-data.json';
+import './starter-kits-style';
+
+// NOTE: The table classes used in this component correspond to
+// those used in the markdown utility. Ideally we will soon create
+// a table component that both the markdown utility and this page
+// can use. This component could even use something like griddle
+// to allow sorting and such.
```

Comment:
If we want something super light, it's enough to attach a little standalone script (these exist for sure) that hooks into the table.